lib/kamal/commands/base.rb

```diff
@@ -11,7 +11,7 @@ def initialize(config)
     end
 
     def run_over_ssh(*command, host:)
-      "ssh #{ssh_proxy_args} -t #{config.ssh.user}@#{host} -p #{config.ssh.port} '#{command.join(" ").gsub("'", "'\\\\''")}'".squish
+      "ssh#{ssh_proxy_args} -t #{config.ssh.user}@#{host} -p #{config.ssh.port} '#{command.join(" ").gsub("'", "'\\\\''")}'".squish
```

Comment:
Hmm i thought i had tests passing with this. But i think with this new approach you can remove squish at the end